openfisca_core/columns.py

```diff
@@ -32,6 +37,7 @@ class Column(object):
     formula_class = None
     is_period_size_independent = False  # When True, value of column doesn't depend from size of period (example: age)
     is_permanent = False  # When True, value of column doesn't depend from time (example: ID, birth)
+    period_behavior = None
```

Comment:
If we don't have a consensus on a better name that `period_unit`, let switch back to this. Search-friendliness matters much less than name meaningfulness.